libs/dh/market-participant/feature-edit-actor/src/lib/market-participant-edit-actor/master-data/dh-market-participant-actor-master-data.component.html

```diff
@@ -23,7 +23,7 @@
       <input
         [disabled]="actor !== undefined"
         wattInput
-        [(ngModel)]="changes.gln"
+        [(ngModel)]="changes.actorNumber"
         (ngModelChange)="onModelChanged()"
         #glnInputModel="ngModel"
```

Comment:
nitpick: change this variable accordingly e.g. `#actorNumberInputModel`. Also on the `watt-error` bellow.